samples/crypto/persistent_key_usage/sample.yaml

```
@@ -1,67 +1,73 @@
 sample:
   description: |
-    This app provides an example of using peristent keys with the
-    PSA APIs. A random AES persistent key is generated and imported
-    to the PSA keystore.
-  name: Persistent key example
+    This sample provides an example of using persistent keys with the
+    PSA Crypto API. A random AES persistent key is generated and used.
+  name: Persistent key usage example
```

No need for "example" or "sample" in `name`. Most samples don't have this.
```suggestion
  name: Persistent key usage
```

---

Oh right. Forgot about that. Got distracted by how it was.